WordPress/src/main/java/org/wordpress/android/ui/reader/repository/ReaderDiscoverRepository.kt

```diff
@@ -63,6 +70,28 @@ class ReaderDiscoverRepository constructor(
         shouldAutoUpdateTagUseCase.stop()
         getPostsForTagWithCountUseCase.stop()
         readerUpdatePostsEndedHandler.stop()
+        postLikeActionUseCase.stop()
+    }
+
+    fun performLikeAction(post: ReaderPost, isAskingToLike: Boolean, wpComUserId: Long) {
+        launch {
+            val event: ReaderRepositoryEvent
+            try {
+                event = postLikeActionUseCase.perform(post, isAskingToLike, wpComUserId)
+            } catch (e: IllegalStateException) {
+                return@launch
```

Comment:
We might want to at least log an exception here. Empty catch statements are considered a bad practice. Wdyt?

Reply:
Sure, no problem, I'll log an event.